app/src/main/java/org/jellyfin/androidtv/details/BaseDetailsFragment.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.details
+
+import android.os.Bundle
+import androidx.leanback.app.DetailsSupportFragment
+import androidx.leanback.widget.*
+
+open class BaseDetailsFragment : DetailsSupportFragment() {
+	protected lateinit var rowsAdapter: ArrayObjectAdapter
+
+	override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Empty function, only calls that only calls it's parent could probably be removed

Reply:
Sure, this is a leftover, but might as well stay there because once we have more sub-classes modeled out, we'll probably move stuff up there as well

Reply:
I removed it now